test/tests/python-hy/container.sh

```diff
@@ -16,13 +16,13 @@ fi
 
 # Hy is complicated, and uses Python's internal AST representation directly, and thus Hy releases usually lag behind a little on major Python releases (and we don't want that to gum up our tests)
 # see https://github.com/hylang/hy/issues/1111 for example breakage
-if ! "$python" -c 'import sys; exit((sys.version_info[0] == 3 and sys.version_info[1] >= 8) or sys.version_info[0] > 3)'; then
+if ! "$python" -c 'import sys; exit((sys.version_info[0] == 3 and sys.version_info[1] >= 10) or sys.version_info[0] > 3)'; then
 	echo >&2 'skipping Hy test -- not allowed on Python 3.8+ (yet!)'
```

Comment:
:eyes:

(also, it looks like https://github.com/hylang/hy/blob/0ec70a2f650c3c2b5dc29b8f0bd1cbe74384f7f4/setup.py#L112-L114 isn't updated, but maybe that's just an oversight?)

Reply:
Filed https://github.com/hylang/hy/pull/1956